spec/overcommit/hook/pre_commit/php_stan_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'spec_helper'
+
+describe Overcommit::Hook::PreCommit::PhpStan do
+  let(:config)  { Overcommit::ConfigurationLoader.default_configuration }
+  let(:context) { double('context') }
+  subject { described_class.new(config, context) }
+
+  before do
+    subject.stub(:applicable_files).and_return(%w[sample.php])
+  end
+
+  context 'when phpstan exits successfully' do
+    before do
+      sample_output = [
+        ''
+      ].join("\n")
```

Comment:
This might be better expressed as just an empty string.

```ruby
sample_output = ''
```

Reply:
👍